model/enduser/common.yaml

```diff
@@ -0,0 +1,22 @@
+groups:
+  - id: identity
+    type: attribute_group
+    brief: >
+      These attributes may be used for any operation with an authenticated and/or authorized enduser.
+    attributes:
+      - ref: enduser.id
+        deprecated: Replaced by `enduser.pseudo.id` attribute.
+        requirement_level: recommended
+      - ref: enduser.pseudo.id
+        requirement_level: recommended
+      - ref: enduser.role
+        deprecated: "Removed."
+        requirement_level: recommended
+      - ref: enduser.scope
+        deprecated: "Removed."
+        requirement_level: recommended
+      - ref: enduser.authentication.id
+        requirement_level: required
+        note: >
+          The `enduser.authentication.id` attribute is intended to provide an unique identifier of an authenticated enduser.
+          The deprecated attributes `enduser.authentication.role` and `enduser.authentication.scope` are removed from the enduser registry.
```

Comment:
```suggestion
```

I don't think that' accurate and also there is no need to mention changes in yaml that have no effect on markdown.